src/components/WebexParticipantRoster/WebexParticipantRoster.jsx

```diff
@@ -5,15 +5,23 @@ import {List} from '@momentum-ui/react';
 import WebexParticipant from '../WebexParticipant/WebexParticipant';
 import useMemberships from '../hooks/useMemberships';
 
+/**
+ * Enum for types of destinations.
+ *
+ * @external DestinationType
+ * @see {@link https://github.com/webex/component-adapter-interfaces/blob/master/src/MembershipsAdapter.js#L21}
+ */
+
 /**
  * Displays the roster of Webex meeting or room participants.
  *
- * @param {string} props.location  the roster of Webex meeting or room participants.
+ * @param {string} props.destinationID  the roster of Webex meeting or room participants.
+ * @param {string} props.destinationType Type of destination of the membership roster
  *
  * @returns {object} JSX of the component
  */
-export default function WebexParticipantRoster({destination}) {
-  const participants = useMemberships(destination);
+export default function WebexParticipantRoster({destinationID, destinationType}) {
```

Comment:
Should we update the component name?! (separate commit?)
```suggestion
export default function WebexMemberRoster({destinationID, destinationType}) {
```

Reply:
Was going to do this in another PR to keep this one focused, but I can

Reply:
Sure, if you wanna do it on another PR that's fine, as long as we're on the same page 😁